docker/Dockerfile-i686

```diff
@@ -0,0 +1,11 @@
+FROM phusion/centos-5-32
+MAINTAINER The ManyLinux project
+
+ENV LC_ALL=en_US.UTF-8
+ENV LANG=en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+ENV PATH /opt/rh/devtoolset-2/root/usr/bin:$PATH
+ENV LD_LIBRARY_PATH /opt/rh/devtoolset-2/root/usr/lib64:/opt/rh/devtoolset-2/root/usr/lib
```

Comment:
Just checked (`docker run --rm -i -t quay.io/pypa/manylinux1_i686 /bin/bash`), and the directory exists, it's just empty.